packages/components/src/LightBox/LightBox.mdx

```diff
@@ -0,0 +1,73 @@
+---
+name: LightBox
+menu: Components
+route: /components/light-box
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Button } from "../Button";
+import { LightBox } from ".";
+
+# Light Box
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+LightBox is a component designed to display an array of images to the user. A
+user clicks on a control and is shown a carousel style presentation of the
+defined images. This component is a wrapper of the frontend-collective's
+react-image-lightbox third party library.
+
+```ts
+import { LightBox } from "@jobber/components/LightBox";
+```
+
+<Playground>
+  {() => {
+    const [open, isOpen] = useState(false);
+    return (
+      <>
+        <Button label="😈 Click me!" onClick={() => isOpen(true)} />
+        <LightBox
+          open={open}
+          images={[
+            {
+              title: "😃 Happy Face",
+              url: "https://i.imgur.com/6Jcfgnp.jpg",
+              caption: "This is a happy face"
+            },
+            {
+              title: "😮",
+              url: "https://i.imgur.com/OsLxYkT.jpg",
+              caption: "Suprised face"
+            },
+            {
+              url: "https://i.imgur.com/uigfzs5.jpg"
+            }
+          ]}
+          onRequestClose={() => isOpen(false)}
+        />
+      </>
+    );
+  }}
+</Playground>
+
+## Design & Usage Guidelines
+
+A lightbox should:
```

Comment:
@chris-at-jobber Wouldn't mind you taking a quick look at the guides here if you have a moment? 😄

Reply:
I'd ideally like to see a bit more direction around the actual use case for it, this feels a bit more like the rationale for why it looks the way it does. @jlelford when you're back you can add any thoughts you might have, maybe something like 

"A lightbox should be used when viewing an image is the user's primary goal" so that we don't end up having the lightbox used for things like in-product upgrade prompts? A bit of disambiguation around when a lightbox vs a modal that contains an image, or something?

Reply:
To be clear, I also don't want to block this from going into Atlantis on doc changes

Reply:
Made some changes here to better reflect your comments.

Reply:
Just got this from Josh, I think this encapsulates it pretty well:
>A lightbox's primary goal is to allow users to see a higher resolution view of images contained within the page they're viewing. The application of a lightbox facilitates smaller thumbnail images to be used in an interface without compromising the detail within.
The lightbox should only be used when displaying an image at a greater scale is the primary function of opening it. Additional content types would be better suited by using a dialogue or alternative method.

Reply:
This is still outstanding.